app/react/kubernetes/components/KubernetesEventsDatatable/columns/date.tsx

```diff
@@ -0,0 +1,13 @@
+import { formatDate } from '@/portainer/filters/filters';
+
+import { columnHelper } from './helper';
+
+export const date = columnHelper.accessor(
+  (event) => formatDate(event.lastTimestamp || event.eventTime),
+  {
+    header: 'Date',
+    id: 'Date',
+    cell: ({ row: { original: event } }) =>
+      formatDate(event.lastTimestamp || event.eventTime),
```

Comment:
we can just use the accessor value, this way if we change it for some reason, the value in the column will change:
```suggestion
    cell: ({ getValue }) =>
      formatDate(getValue()),
```

Reply:
Nice suggestion, updated